docs/src/pages/discover-more/showcase/appList.js

```diff
@@ -690,20 +683,12 @@ const appList = [
     link: 'https://justaskusers.com/',
     index: 24,
   },
-  {
-    title: 'Gadr',
-    description:
-      'Follow top performing crypto traders. ' +
-      'Get notifications from your favorite traders. Get a full overview of your crypto assets.' +
-      'Share your portfolio with the world and see how you stack up to the best',
-    link: 'https://gadr.io/',
-    index: 33,
-  },
```

Comment:
What was the issue with this one?

Reply:
It's yet another crypto project. It's displayed at the top when ranked by "Newest".

Reply:
Also, we don't have any screenshot.